### PR TITLE
community resource edit: added the edit functionality to the present …

### PR DESCRIPTION
**Pull request by Anwesha2**
…master branch files

### CODE REVIEW

File: app/api.py [hunk 1]
```diff
@@ -75,6 +75,84 @@ def post_communityresource_register(body):
     return resource.to_dict(), 200
 
 
+def put_community_resource_edit(body):
+
+    geolocator = Nominatim() # will copy over from master branch
```
Comment (abetlen) [inline app/api.py]: The geopy module should be wrapped in some kind of service module as it requires you to make an http request to a 3rd party service every time you perform a lookup. Using geopy directly can be problematic in testing, we should have something similar to the manager classes that interact with the database.

File: app/api.py [hunk 2]
```diff
@@ -75,6 +75,84 @@ def post_communityresource_register(body):
...
+    try:
+        CommunityResourceManager.edit_community_resource(int(body["number"]), body["name"], lat, lon, body["contact_name"], body["email"], body["phone_number"]) 
+    except NoExistingCommunityResource:
+        return NoContent, 500
+    except InvalidCommunityResourceInfo:
+        return NoContent, 500
+    return NoContent, 200
+
+
+def put_community_resource_edit_name(body):
```
Comment (abetlen) [inline app/api.py]: It's preferable to use a single put for a resource. You can choose to only update the fields included in the body for partial updates, or require the client to make a full get request first and change the values they want to update.

File: app/api.yml [hunk 3]
```diff
@@ -207,6 +207,259 @@ paths:
                 type: boolean
       security:
         - basic: []
+  /communityresource/edit:
```
Comment (abetlen) [inline app/api.yml]: REST endpoints should not include verbs.

Change this to just
```
PUT /communityresource/<id>
```

File: app/api.py (hunk 2 above)
Comment (Anwesha2) [reply app/api.py]: I'll default to the latter option

File: app/api.py (hunk 1 above)
Comment (AlisonT1230) [reply app/api.py]: Does wrapping in a service module mean putting it in a separate private function (as suggested in code_review.txt)?  Or could it be a static function under the manager class?

**Pull request merged**